web-api/terraform/template/api.tf

```diff
@@ -24,26 +43,28 @@ resource "aws_lambda_function" "api_lambda" {
   }
 }
 
-resource "aws_lambda_function" "api_clamav_lambda" {
-  filename      = data.archive_file.zip_api.output_path
-  function_name = "api_clamav_${var.environment}"
-  role          = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/lambda_role_${var.environment}"
-  handler       = "api.handler"
-  source_code_hash = data.archive_file.zip_api.output_base64sha256
-  timeout = "29"
-  memory_size = "3008"
+# TODO: The virus definitions have gotten too large to live on Lambda... find alternative solution to get 
```

Comment:
Is there a user story for this work?

Reply:
Yes: https://app.zenhub.com/workspaces/flexionef-cms-5bbe4bed4b5806bc2bec65d3/issues/flexion/ef-cms/6053